bundles/org.openhab.binding.electroluxappliance/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -448,4 +492,238 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="appliance-running">
+		<item-type>Switch</item-type>
+		<label>Powered On</label>
+		<description>The device's state running state.</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="ambient-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Ambient Temperature</label>
+		<description>The measured ambient temperature.</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f %unit%">
+		</state>
+	</channel-type>
+
+	<channel-type id="target-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>The target set-point temperature.</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state min="16" max="32" step="1" pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="sleep-mode">
+		<item-type>Switch</item-type>
+		<label>Sleep Mode</label>
+		<description>Whether sleep mode is active.</description>
+		<category>Switch</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="fan-swing">
+		<item-type>Switch</item-type>
+		<label>Fan Swing</label>
+		<description>Whether fan swing is active.</description>
+		<category>Flow</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="child-ui-lock">
+		<item-type>Switch</item-type>
+		<label>Child Lock</label>
+		<description>Whether child lock is active.</description>
+		<category>Lock</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="fan-mode">
+		<item-type>String</item-type>
+		<label>Fan Speed</label>
+		<description>The fan speed mode.</description>
+		<category>Flow</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
```

Comment:
```suggestion
			<tag>Control</tag>
			<tag>Airflow</tag>
```

Reply:
Thank you I missed that - am still getting used to the tags. Should be added now.